core/controller/src/main/scala/org/apache/openwhisk/core/loadBalancer/ShardingContainerPoolBalancer.scala

```diff
@@ -265,6 +268,9 @@ class ShardingContainerPoolBalancer(
     chosen
       .map { invoker =>
         setupActivation(msg, action, invoker)
+        logging.info(
+          this,
+          s"activation ${msg.activationId} of $actionType-action '${msg.action.asString}' invoked by namespace '${msg.user.namespace.name.asString}' with memory-limit of ${action.limits.memory.megabytes} MB will be executed on ${invoker.instance}")
```

Comment:
You are adding a new message to be logged for each activation while method `sendActivationToInvoker()` called in next line logs the activation anyway (https://github.com/apache/incubator-openwhisk/pull/4159/files#diff-f749d7d6e4e8ae0b5202a75af1136202R345).

I suggest to extend the existing message - the current proposal may have a performance impact because of the additional log messages. I understand that extending the existing message would require a change of parameters passed to `sendActivationToInvoker()`.

Reply:
I agree, that there is already a message below, but we do not have the information about memory-limit and action-kind there.
And I think it does not make sense to pass these parameters to this method only to be able to write logs.
But my suggestion would be to remove the logmessage there to avoid duplication and only keep the marker there.